### ISSUE TITLE
Remove lpri deps

### ISSUE THREAD

**Issue by yrsegal** [yrsegal/LaserQueue#144]
Make priorities make sense
this will mess up any existing queues, sorry

affects both the backend and the frontend

**Comment by yrsegal** [comment]
Rules for priorities:

```
  [
    [
      {job}, {job}, ...
    ] for each priority
  ]
```

0-x, x being highest priority possible.
so 0, (length of the 0 priority list -1) would be the lowest possible, and 5, 0 would be highest possible.

why did I ever let @sdaitzman convince me to make it otherwise

**Comment by yrsegal** [comment]
closed by c37e770526b7835deec63061b0e789c2503eda7b
